### DISCUSSION TITLE
Reads correct values only when logged in

### DISCUSSION THREAD

**Discussion by kylemonti** [jomjol/AI-on-the-edge-device#2889] [General]
Something weird is going on with my setup (and I'm pretty sure I'm missing something). I set up 2 ESP32CAM and they're running the latest firmware (15.6). I watch them work via the browser on my PC and everything looks fine. I turn off my PC or go do something else, log back in a couple of hours later, and get the error that it's reading a negative difference (ie. the previous number was too high). Turns out one of the 6's on the meters is being read as 8, but only when I'm not watching over them with the browser. When I am connected, they read correctly, then I log off and they start reading incorrectly. I log back in, they read correctly again and that's when the error pops up. 

One more small annoyance is that every time I reset the "previous value", it shows on the Overview that it's there, but after it takes the next reading, it forgets about it and goes to the previous "previous value" i.e. the one that it read wrong. 

It can't be that me dialling in via my browser affects what it's reading, right?

**Comment by caco3** [comment]
no, it can't be your webbrowser :)

By any change, you are not powering it from the PC, causing it to be stopped being powered?

Please provide some screenshots of the ROIs and a log covering the time before leaving until coming back.

**Comment by kylemonti** [comment]
I must have done something dumb somewhere...
No they are both powered in parallel by a Samsung USB phone charger. The charger is barely getting warm, which tells me they're have more power if they needed it. 

Anyways, I've attached a screenshot and 1 log from each of the ESP32 modules for the same day. You'll see it clearly where it starts saying "negative read". It was mostly confusing the digits 6 and 8 on the red dials.

![Screenshot from 2024-02-13 15-11-53](https://github.com/jomjol/AI-on-the-edge-device/assets/44883186/907d1271-5dd8-4867-b72e-a208749e5d64)
[log_2024-02-11HW.txt](https://github.com/jomjol/AI-on-the-edge-device/files/14267791/log_2024-02-11HW.txt)
[log_2024-02-11KW.txt](https://github.com/jomjol/AI-on-the-edge-device/files/14267794/log_2024-02-11KW.txt)

**Reply by kylemonti** [reply]
PS. I am getting around the issue right now by turning on "negative values", knowing that when I look at the graph, I can ignore the upwards spikes. Otherwise it goes crazy :)
